src/test/java/com/ericsson/ei/queryservice/test/QueryServiceRESTAPITest.java

```diff
@@ -81,7 +79,7 @@
     private QueryAggregatedObjectControllerImpl aggregatedObjectController;
 
     @MockBean
-    private QueryMissedNotificationControllerImpl missedNotificationController;
+    private FailedNotificationControllerImpl failedNotificationsController;
```

Comment:
Same comment for this test class also...

Reply:
done